frontend/src/LogOut.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+
+import {
+    Redirect
+} from "react-router-dom";
+
+
+export default function LogOut({setToken}) {
+
+    const [loggedOut, setLoggedOut] = useState(false);
+
+    function logout() {
+        sessionStorage.clear('acces_token');
+        setLoggedOut(true);
+        setToken("");
+
+    };
+
+    if (loggedOut) {
+        return <Redirect to="/" push={true} />
```

Comment:
Ezt szerintem az App routingnál kellene eldönteni a token alapján.

Reply:
Töröltem innen. App.js-ben egyébként is volt token ellenőrzés.